Documentation/ContentObjects/UserAndUserInt/_ExampleListRecords.php

```diff
@@ -5,15 +5,24 @@
 namespace MyVendor\SitePackage\UserFunctions;
 
 use Psr\Http\Message\ServerRequestInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
 use TYPO3\CMS\Core\Database\ConnectionPool;
-use TYPO3\CMS\Core\Utility\GeneralUtility;
 use TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer;
 
 /**
  * Example of a method in a PHP class to be called from TypoScript
+ *
+ * The class is defined as public as we use dependency injection in
+ * this example. If you do not need dependency injection, the
+ * "Autoconfigure" attribute should be omitted!
```

Comment:
What do you think about:

```suggestion
 * This class is called via GeneralUtility::makeInstance. That's why you **should** mark 
 * it as `public: true`. When using dependency injection, like in this example, you
 * **have to** mark the class as `public: true`.
```

Reply:
I disagree, there is no need to mark a user function as public as long as you don't use DI.